src/mlpack/bindings/R/mlpack/DESCRIPTION.in

```diff
@@ -0,0 +1,25 @@
+Package: mlpack
+Title: 'Rcpp' Integration for the 'mlpack' Library
+Version: @PACKAGE_VERSION@
+Date: @PACKAGE_DATE@
+Author: mlpack Team
+Maintainer: mlpack Team <mlpack-git@lists.mlpack.org>
```

Comment:
Should I change the Maintainer to: `Ryan Curtin <ryan@ratml.org>`, as auto-generated `DESCRIPTION` file will show Maintainer as: `mlpack Team <mlpack-git@lists.mlpack.org>`.

Reply:
Per CRAN Policy it ought to be an individual address anand not a list.

So, yay, Ryan it is.

Reply:
I will make the changes. Thank You :)

Reply:
Done.

Reply:
@Yashwants19 if you like, you can do it.  I am already the maintainer of too many things. :)  I can do it if needed though---but as is probably clear from this whole summer, I don't know all that much about R and I know even less about CRAN. :smile: